src/main/g8/build.sbt

```diff
@@ -8,14 +8,13 @@ lazy val root = project
   .settings(moduleName := "root")
   .settings(noPublishSettings: _*)
   .settings(scalaMetaSettings: _*)
+  .settings(libraryDependencies ++= commonDeps ++ freestyleCoreDeps())
```

Comment:
These settings should be placed in crossDepSettings because of Scala's cross-compilation.